pkg/apiserver/api_installer.go

```diff
@@ -321,10 +321,12 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 
 			// list or post across namespace.
 			// TODO: more strongly type whether a resource allows these actions on "all namespaces" (bulk delete)
-			namer = scopeNaming{scope, a.group.Linker, gpath.Join(a.prefix, itemPath), true}
-			actions = appendIf(actions, action{"LIST", resource, params, namer}, isLister)
-			actions = appendIf(actions, action{"POST", resource, params, namer}, isCreater)
-			actions = appendIf(actions, action{"WATCHLIST", "watch/" + resource, params, namer}, allowWatchList)
+			if !hasSubresource {
```

Comment:
I think we might need this, but I can't remember where.  Will look it up in the morning - please hold till then.